examples/spark/pytorch/pytorch_lightning_spark_mnist.py

```diff
@@ -184,6 +188,14 @@ def on_train_end(self, trainer, model):
                                    verbose=True,
                                    mode='max'))
 
+    if False:
```

Comment:
What is the reason of hard-coding `False` here?

Reply:
That was a miss from testing, fixed